lib/facter/gce/metadata.rb

```diff
@@ -45,7 +45,7 @@ def with_metadata_server
 
         begin
           Timeout.timeout(timeout) do
-            body = open(@url, :proxy => nil).read
+            body = open(@url, options={:proxy => nil, "Metadata-Flavor" => "Google"}).read
```

Comment:
I think `option=` is unnecessary, as the `open` command accepts a hash as second argument.
https://ruby-doc.org/stdlib-2.6.3/libdoc/open-uri/rdoc/OpenURI.html